mne/event.py

```diff
@@ -633,17 +655,28 @@ def find_events(raw, stim_channel=None, output='onset',
     return events
 
 
-def _mask_trigs(events, mask):
+def _mask_trigs(events, mask, mask_type):
     """Helper function for masking digital trigger values"""
-    if not isinstance(mask, int):
-        raise TypeError('You provided a(n) %s. Mask must be an int.'
-                        % type(mask))
+    if mask is not None:
+        if not isinstance(mask, int):
```

Comment:
slightly offtopic - I think ability to pass `mask` as a boolean array would be quite intuitive (more intuitive maybe).